docs/xks/developer-guide/security.md

```diff
@@ -9,3 +9,54 @@ Certain policies will always be enforced in the cluster as a guard rail to minim
 any request to create or update a Kubernetes resource will first have to pass through the policy check. If the resource in question does not comform to the policy it will be rejected
 by the API server when applying the change. Knowing this is important as certain features or options documented on the Internet may not be available or are restricted in the XKS service.
 This can include things like certain types of volume mounts, labeling requirements or container capabilities for a Pod.
+
+## Security Features
+
+### OPA Gatekeeper
+
+OPA (Open Policy Agent) Gatekeeper is a product in which we have a set of rules defining how we allow containers to be created. To simplify this for the developers these rules are then applied automatically to all Pods created to achive our set standard.
```

Comment:
Isn't this what the previous section already says, except that it does not mention OPA? If we want to mention that we are using OPA, I think the section above should be updated.